docs/getting_started/online_comp_getting_started.md

```diff
@@ -75,10 +75,14 @@ The output panel also displays results for the "Find in Folder" functionality. H
 
 ## Create your first program
 
-From the main toolbar, click the "New" button, and complete the program name field in the dialog box like on the image below.
+From the main toolbar, click the "New" button, select your target platform and choose a starter template from the drop-down list or select "Empty Program" to create a blank project. Complete the "Program Name" field like in the image below.
```

Comment:
- Please change the first sentence to "From the main toolbar, click the `New` button, and select your target platform. Choose a starter template from the drop-down list, or select `Empty Program` to create a blank project." for formatting and style.

- Please change the double quotes on each side of "Program Name" to the ` symbol.